app/gui/src/project-view/util/data/vec2.ts

```diff
@@ -60,22 +60,22 @@ export class Vec2 {
     return Math.abs(this.x - other.x) < epsilon && Math.abs(this.y - other.y) < epsilon
   }
 
-  /** TODO: Add docs */
+  /** Check if both coords are 0. */
   isZero(): boolean {
     return this.x === 0 && this.y === 0
   }
 
-  /** TODO: Add docs */
+  /** Check if both coords have finite values. */
   isFinite(): boolean {
     return Number.isFinite(this.x) && Number.isFinite(this.y)
   }
 
-  /** TODO: Add docs */
+  /** Change Inf values in vector to 0. */
```

Comment:
`NaN` is also non-finite

Reply:
This method intentionally coerces `NaN` to `0` as well. I think it's just a matter of adjusting docs.

```suggestion
  /** Change any infinite or non-numeric (e.g. `Infinity` or `NaN`) vector values to `0`. */
```

Reply:
Yes

Reply:
Yes